glusterd2-e2e-github-pr.yml

```diff
@@ -1,57 +1,54 @@
 - job:
-    name: 'gluster_glusterd2'
+    name: 'gluster_glusterd2_e2e'
     node: gluster
-    description: Verify GlusterD2 pull-requests
+    description: Runs e2e tests against glusterd2 pull requests
     project-type: freestyle
-
+    concurrent: true
 
     scm:
     - git:
-        branches:
-        - '*/master'
-        git-tool: Default
         url: https://github.com/gluster/glusterd2.git
+        branches:
+        - origin/master
     - git:
         url: https://github.com/gluster/centosci.git
         branches:
         - origin/master
 
-
     builders:
-    - shell: - shell: !include-raw scripts/common/jenkins-job.py
-
-    parameters:
-    - string:
-        default: scripts/common/run-test.sh
-        description: Test script to execute on the reserved machine.
-        name: TEST_SCRIPT
-
+    - shell: !include-raw: scripts/common/get-node.sh
+    - shell: !include-raw: scripts/common/rsync.sh
+    - shell: scripts/common/bootstrap.sh $WORKSPACE/extras/centos-ci.sh
```

Comment:
This is the part I'm not sure who clones the repo and pulls the PR that triggered the job.